modules/ding_news/ding_news.features.inc

```diff
@@ -52,13 +45,6 @@ function ding_news_image_default_styles() {
         'weight' => 0,
       ),
       23 => array(
-        'label' => 'Scale',
-        'help' => 'Scaling will maintain the aspect-ratio of the original image. If only a single dimension is specified, the other dimension will be calculated.',
-        'effect callback' => 'image_scale_effect',
-        'dimensions callback' => 'image_scale_dimensions',
-        'form callback' => 'image_scale_form',
-        'summary theme' => 'image_scale_summary',
-        'module' => 'image',
```

Comment:
Why is this information removed?

Reply:
The new features version removes this, my guess is this is redundant information.

Reply:
okay